### PR TITLE
Added AttenuationTest to render-fidelity-tools tests

### PR DESCRIPTION
**Pull request by augmentedsky**
Added AttenuationTest model and its golden files to render-fidelity-tools tests

### CODE REVIEW

**Comment by elalish** [comment]
I just disabled the broken test runner. Please pull master and let's see if we can get these tests to pass.

File: packages/render-fidelity-tools/test/config.json [hunk 1]
```diff
@@ -75,6 +75,23 @@
...
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/AttenuationTest/glTF/AttenuationTest.gltf",
+      "dimensions": {
+        "width": 512,
+        "height": 1024
+      },
+      "target": {
+        "y": 3.6
+      },
+      "orbit": {
+        "radius": 11.2
```
Comment (elalish) [inline packages/render-fidelity-tools/test/config.json:89]: I would start with the default values (just removing these listings), then add back in the ones you need until the model is nicely framed. Only use custom dimensions if the model has a really odd aspect ratio.

**Review by elalish — CHANGES_REQUESTED** [review]
Good start!

File: packages/render-fidelity-tools/test/config.json (hunk 1 above)
Comment (augmentedsky) [reply packages/render-fidelity-tools/test/config.json]: Sorry I just saw this. Let me update them in the next commit

Comment (augmentedsky) [reply packages/render-fidelity-tools/test/config.json]: Just sent another commit let me know what you think

**Review by elalish — APPROVED** [review]
I'm not sure why Github's rich diff tool seems to be not working. Would be great if you could paste one of your goldens into a comment so we can see.

**Comment by elalish** [comment]
@googlebot I signed it!

**Comment by augmentedsky** [comment]
Here are the attenuationtest screenshots. How do they look?

babylon-golden
filament-golden
gltf-sample-viewer-golden
model-viewer-golden

**Comment by timmmeh** [comment]
Based on the Don McCurdy viewer https://gltf-viewer.donmccurdy.com/ this looks reasonable

**Pull request merged**